Tests/Client/Unit Tests/WebSocket/WebSocketTests.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  WebSocketTests.swift
+//  StreamChatClientTests
+//
+//  Created by Alexey Bukhtin on 02/05/2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+@testable import StreamChatClient
+
+final class WebSocketTests: XCTestCase {
+    
+    let mockProvider = WebSocketProviderMock(request: URLRequest(url: URL(string: "http://test.com")!), callbackQueue: .main)
+    let logger = ClientLogger(icon: "🦄", level: .info)
+    
+    override static func setUp() {
+        WebSocket.pingTimeInterval = 2
+    }
+    
+    override class func tearDown() {
+        WebSocket.pingTimeInterval = 25
+    }
```

Comment:
I don't understand what's happening here and why.

Reply:
I'll add comments.